resources/erlang_zypper_repository_on_suse.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+#
+# Cookbook Name:: rabbitmq
+# Resource:: erlang_zypper_repository_on_suse
```

Comment:
The naming we use for these resources is `erlang_[type]_repository_on_[hosting service]` :)